deploy/helm/cht-chart/templates/couchdb-claim0-persistentvolumeclaim.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if and (not .Values.couchdb.clusteredCouch_enabled) (eq .Values.environment "remote") }}
```

Comment:
@henokgetachew What about `local` environment installations? They would also need a PV and PVC for database files to be written & read.

Reply:
This change has been performed for the clustered setup already. It will be performed for the single-node setup next.